src/game.js

```diff
@@ -0,0 +1,13 @@
+function Game() {
+  this.frames = [];
+  this.scores = [];
+  this.score = 0;
+  this.frameCount = 0;
+}
+
+Game.prototype.playFrame = function(frame) {
+  var frame = new Frame;
```

Comment:
'frame' is already defined.<br>Missing '()' invoking a constructor.<br>'Frame' is not defined.